segmentstore/server/src/main/java/io/pravega/segmentstore/server/store/ServiceBuilder.java

```diff
@@ -122,8 +124,10 @@ private ServiceBuilder(ServiceBuilderConfig serviceBuilderConfig, ServiceConfig
 
         // Setup Thread Pools.
         String instancePrefix = getInstanceIdPrefix(serviceConfig);
-        this.coreExecutor = executorBuilder.apply(serviceConfig.getCoreThreadPoolSize(), instancePrefix + "core");
-        this.storageExecutor = executorBuilder.apply(serviceConfig.getStorageThreadPoolSize(), instancePrefix + "storage-io");
+        this.coreExecutor = executorBuilder.apply(serviceConfig.getCoreThreadPoolSize(), instancePrefix + "core", Thread.NORM_PRIORITY);
+        this.storageExecutor = executorBuilder.apply(serviceConfig.getStorageThreadPoolSize(), instancePrefix + "storage-io", Thread.NORM_PRIORITY);
+        this.lowPriorityExecutor =
```

Comment:
What does 2 mean here? Above you use `Thread.NORM_PRIORITY`. Is there another constant you can use?

Reply:
Also do not hardcode the number of threads. Please add an entry in `ServiceConfig`, default it to 10, so that we may change it at any time if we encounter problems.

Reply:
Sure, done.